### PR TITLE
reset password schema for API docs swagger updated #1434

### LINKED ISSUE #1434: Update api schema for password reset endpoints
Override the default forgot password and password reset endpoints and:
- [ ] Add relevant fields to serializers
- [ ] update test cases

### PR DESCRIPTION
**Pull request by itxsoumya**
- fixes #1434 

## Proposed Changes
Wrote custom Schema for password reset endpoints.

### CODE REVIEW

File: care/users/reset_password_views.py [hunk 1]
```diff
@@ -49,7 +49,36 @@ class ResetPasswordCheck(GenericAPIView):
...
+
+        examples=[
+            OpenApiExample(
+
+                name='Password Reset Check Example',
+                summary='Password Reset Check',
+                description='Provide your unique token',
+                value={
+                    "token": "your token"
+                },
```
Comment (sainak) [inline care/users/reset_password_views.py:64]: this needs to be defined as parameters.
https://drf-spectacular.readthedocs.io/en/latest/customization.html#step-2-extend-schema

do the same for other methods

File: care/users/reset_password_views.py
Comment (sainak) [inline care/users/reset_password_views.py:1]: https://github.com/coronasafe/care/blob/master/CONTRIBUTING.md
make sure the code is properly formatted

File: care/users/reset_password_views.py (hunk 1 above)
Comment (itxsoumya) [reply care/users/reset_password_views.py:64]: Do we have to define the QuerySerializer or it by default provides us.

**Comment by itxsoumya** [comment]
@sainak I have updated the parameters for all password reset routes


Screenshot from 2024-03-02 08-33-46

**Comment by itxsoumya** [comment]
@sainak review please

**Comment by vigneshhari** [comment]
@sainak bump!

**Review by sainak — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**